javascript/javascript3/week3/promise.js

```diff
@@ -0,0 +1,34 @@
+//Getting into promises
+function userRepositories(userName) {
+  const res = fetch(
+    `https://api.github.com/search/repositories?q=user:${userName}`
+  );
+  res
+    .then((response) => response.json())
+    .then((data) => {
+      console.log(data);
+      const ul = document.createElement("ul");
+      ul.innerHTML = `${userName}'s repositories`;
+      document.body.append(ul);
+      console.log(data.items);
+      data.items.forEach((render) => {
+        const liId = document.createElement("li");
+        const li = document.createElement("li");
+        const li1 = document.createElement("li");
+        liId.innerHTML = `Id: ${render.id}`;
+        li.innerHTML = `Full name: ${render.full_name}`;
+        li1.innerHTML = `Url: ${render.url}`;
+        ul.append(liId);
+        ul.append(li);
+        ul.append(li1);
+      });
```

Comment:
You can also append all together like `ul.append(liId,li,li1)`.
  It is good that you name variable ` liId`  for Id values but it would be better if you name the rest of the const variables likewise, expressing their purpose, something like li to liName, li1 to liUrl.